qiskit_experiments/library/randomized_benchmarking/data/generate_clifford_data.py

```diff
@@ -0,0 +1,188 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+This file is a stand-alone script for generating the npz files in
+:mod:`~qiskit_experiment.library.randomized_benchmarking.clifford_utils.data` directory.
+
+The script relies on the values of ``_CLIFF_SINGLE_GATE_MAP_2Q``
+in :mod:`~qiskit_experiment.library.randomized_benchmarking.clifford_utils`
+so they must be set correctly before running the script.
+"""
+import itertools
+
+import numpy as np
+import scipy.sparse
+
+from qiskit.circuit import QuantumCircuit
+from qiskit.circuit.library import (
+    IGate,
+    HGate,
+    SXdgGate,
+    SGate,
+    XGate,
+    SXGate,
+    YGate,
+    ZGate,
+    SdgGate,
+    CXGate,
+    CZGate,
+)
+from qiskit.quantum_info.operators.symplectic import Clifford
+from qiskit_experiments.library.randomized_benchmarking.clifford_utils import (
+    CliffordUtils,
+    _CLIFF_SINGLE_GATE_MAP_1Q,
+    _CLIFF_SINGLE_GATE_MAP_2Q,
+)
+
+NUM_CLIFFORD_1Q = CliffordUtils.NUM_CLIFFORD_1_QUBIT
+NUM_CLIFFORD_2Q = CliffordUtils.NUM_CLIFFORD_2_QUBIT
+
+
+def _hash_cliff(cliff):
+    """Produce a hashable value that is unique for each different Clifford.  This should only be
+    used internally when the classes being hashed are under our control, because classes of this
+    type are mutable."""
+    table = cliff.table
```

Comment:
I had thought it'd be better to use old `table` to care users of old Terra but I had second thought. This script is not public to users since it is intended to be used only by developers. So it would make sense to assume terra>=0.22 and use new `tableau` here.

Reply:
Changed to use `tableau` in [3cd4324](https://github.com/Qiskit/qiskit-experiments/pull/982/commits/3cd432444e8b8cb7493662c45ddebfa11e92cd00)